[2023-09-23T16:44:10.358+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_test_dag.task_2 manual__2023-09-23T16:43:52.893026+00:00 [queued]>
[2023-09-23T16:44:10.365+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_test_dag.task_2 manual__2023-09-23T16:43:52.893026+00:00 [queued]>
[2023-09-23T16:44:10.365+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-23T16:44:10.386+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): task_2> on 2023-09-23 16:43:52.893026+00:00
[2023-09-23T16:44:10.389+0000] {standard_task_runner.py:57} INFO - Started process 18 to run task
[2023-09-23T16:44:10.391+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'simple_test_dag', 'task_2', 'manual__2023-09-23T16:43:52.893026+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpql52yvag']
[2023-09-23T16:44:10.393+0000] {standard_task_runner.py:85} INFO - Job 68: Subtask task_2
[2023-09-23T16:44:10.411+0000] {logging_mixin.py:151} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-23T16:44:10.435+0000] {task_command.py:415} INFO - Running <TaskInstance: simple_test_dag.task_2 manual__2023-09-23T16:43:52.893026+00:00 [running]> on host simple-test-dag-task-2-ggfk1z9c
[2023-09-23T16:44:10.520+0000] {pod_generator.py:539} WARNING - Model file  does not exist
[2023-09-23T16:44:10.584+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='simple_test_dag' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2023-09-23T16:43:52.893026+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-23T16:43:52.893026+00:00'
[2023-09-23T16:44:10.584+0000] {logging_mixin.py:151} INFO - task 2
[2023-09-23T16:44:10.585+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-23T16:44:10.594+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=simple_test_dag, task_id=task_2, execution_date=20230923T164352, start_date=20230923T164410, end_date=20230923T164410
[2023-09-23T16:44:10.644+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-23T16:44:10.676+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
